tests/integration/test_delete_fast.py

```
@@ -0,0 +1,59 @@
+    )
+    # The old graceful teardown took ~30s; the fast destroy path is ~1-2s. A
+    # 10s ceiling decisively catches a regression to the old behavior while
+    # tolerating CI/dev-machine variance and (for FC) SSH round-trips.
+    assert elapsed < 10, (
+        f"delete of a running shed took {elapsed:.1f}s — expected the fast "
+        f"destroy path (<10s); a regression to the graceful ~30s teardown?"
+    )
+
+    # The shed must actually be gone (a fast delete that didn't delete is worse
+    # than a slow one).
+    listing = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)